CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_zendesk v0.12.0
+This PR updates the `dbt-date` version to the most current, [">=0.9.0", "<0.10.0"].
```

Comment:
Shouldn't this be in it's own `## Updates` section?

Reply:
true, updated